client/driver/fabfile.py

```diff
@@ -105,15 +108,24 @@ def run_oltpbench():
 
 @task
 def run_oltpbench_bg():
-    cmd = "./oltpbenchmark -b {} -c {} --execute=true -s 5 -o outputfile &".\
+    cmd = "./oltpbenchmark -b {} -c {} --execute=true -s 5 -o outputfile > "\
+          "/home/shulij/ottertune/client/driver/oltp.log 2>&1 &".\
```

Comment:
Do not hardcode the log path. You may want to add a ottertune home path in the driver_config.json, and use the parameter.  e.g. CONF['ottertune_home']/client/driver/oltp.log